### DISCUSSION TITLE
ATT&CK Tactic & MBC Objective IDs

### DISCUSSION THREAD

**Discussion by S-C-M-C** [mandiant/capa#1906] [Q&A]
Is there a way to have CAPA display the ATT&CK Tactic & MBC Objective IDs, in addition to displaying the Technique and Behavior IDs?

I am just getting started with CAPA so my apologies if this is an obvious and I just missed it.

**Comment by mr-tz** [comment]
Can you share an example of what you want to see vs. what you get? All data that is shown comes from the files in https://github.com/mandiant/capa-rules.

**Comment by S-C-M-C** [comment]
Thank you.

That helped me track it down.  I now see were it is coded in each rule. 

I was hoping it was a flag or option I was overlooking that would Triger out put to to include the tactics ID code for example : 

Outputing

+------------------------+--------------------------------------------------------------------------------+
| ATT&CK Tactic          | ATT&CK Technique                                                               |
|------------------------+--------------------------------------------------------------------------------|
| DEFENSE EVASION [TA0005]       | Obfuscated Files or Information [T1027] 


Vs what it is

+------------------------+--------------------------------------------------------------------------------+
| ATT&CK Tactic          | ATT&CK Technique                                                               |
|------------------------+--------------------------------------------------------------------------------|
| DEFENSE EVASION        | Obfuscated Files or Information [T1027]
